### PR TITLE
fix(web): datetime display and add TZ into environment

### PR DESCRIPTION
**Pull request by panoti**
All of uploaded photos may have wrong date (creation date, modification date, ...) because EXIF not contain the timezone in its date data. Therefore, Immich worker (name as microservice) will use current container timezone (Etc/UTC) to parse date from string.

For further work, build a plugin/addon to interpolate timezone from geolocation per photo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,6 +146,7 @@ wget -O .env https://raw.githubusercontent.com/immich-app/immich/main/docker/.en
 * Populate `UPLOAD_LOCATION` as prefered location for storing backup assets.
 * Populate a secret value for `JWT_SECRET`, you can use this command: `openssl rand -base64 128`
 * [Optional] Populate Mapbox value to use reverse geocoding.
+* [Optional] Populate `TZ` as your timezone, default is `Etc/UTC`. Immich uses your configured timezone to parse date in metadata (eg: `dateTimeOriginal`, `modifyDate`, ...) per photo. You can use `curl -s http://geoip.ubuntu.com/lookup | grep -oP '(?<=<TimeZone>)(.*)(?=</TimeZone>)'` or `curl -s http://ip-api.com/json/ | grep -oP '(?<=timezone":")(.*?)(?=")'` to fetch your current timezone.
```
Comment (alextran1502) [inline README.md:149]: Probably better to have the user go to the link `http://geoip.ubuntu.com/lookup` and look for timezone, the curl command doesn't work on MacOs

**Comment by alextran1502** [comment]
Do you have two files that I can test?

File: README.md (hunk 1 above)
Comment (panoti) [reply README.md:149]: For macos, I will find another cmd.

**Comment by panoti** [comment]
> Do you have two files that I can test?

You and I are in different timezone, so it's really hard to reproduce my error on your computer. I have a picture taken at 13/07/2022 06:00 PM +7. After uploading to immich, it stored in db as 13/07/2022 06:00 PM +0 (because of the reason mentioned above). Then, I went to the web app, it was shown with 14/07/2022 06:00 PM+7. In this PR, I wanna use correct timezone because we use timestampz type for date. Of course, this PR not fix for the pics taken at multiple timezones, new PR will be neccessary.

File: README.md (hunk 1 above)
Comment (alextran1502) [reply README.md:149]: Probably better to make it clear that the TZ is for displaying the correct time zone on the web.

Comment (panoti) [reply README.md:149]: @alextran1502 Can you help me correct this description by your word?

**Pull request merged**